include/SFML/System/Clock.hpp

```diff
@@ -134,15 +192,18 @@ class SFML_SYSTEM_API Clock
     /// This function acts as a utility for converting clock
     /// duration type instance into sf::Time
     ///
+    /// \param duration std::chorno's duration to be converted
```

Comment:
Typo, should be `std::chrono`

Reply:
I misstyped `std::chrono` as `std::chorno`

Reply:
What they look soo the same 😂